kafka-rest/src/main/java/io/confluent/kafkarest/v2/KafkaConsumerManager.java

```diff
@@ -199,34 +200,18 @@ public String createConsumer(String group, ConsumerInstanceConfig instanceConfig
           );
       }
 
-      Consumer consumer = null;
+      Consumer consumer;
       try {
         if (consumerFactory == null) {
           consumer = new KafkaConsumer(props);
```

Comment:
Can we add the logic to strip `consume.request.timeout.ms` from these properties? It would be better to hard-code a reasonable default than to allow the current misuse.